frontend/package.json

```diff
@@ -113,6 +113,7 @@
       "jest-watch-typeahead": "1.1.0",
       "msw": "0.49.2",
       "next-transpile-modules": "9.0.0",
+      "playwright-msw": "^2.1.0",
```

Comment:
What versioning should we use for `playwright-msw`? For `playwright-msw`, we have it set to the exact version (`"1.29.1"` without caret).

Reply:
Ahh yeah 😓 , I am wondering if we can create a linter to automate this for us or at least throw an error 🤔

Reply:
Addressed in [9bad1f1](https://github.com/iFixit/react-commerce/pull/1319/commits/9bad1f173891203d3a286cbf49a2cf7581013d7f)